nucliadb_sdk/nucliadb_sdk/v2/sdk.py

```diff
@@ -371,6 +371,15 @@ def _check_response(self, response: httpx.Response):
         response_type=ResourceUpdated,
         docstring=docstrings.UPDATE_RESOURCE,
     )
+    update_resource_by_slug = _request_builder(
+        name="update_resource_by_slug",
+        path_template="/v1/kb/{kbid}/slug/{slug}",
+        method="PATCH",
+        path_params=("kbid", "slug"),
```

Comment:
I think you'll need to change it to this unfortunately since we have some duplicate params

```suggestion
        path_template="/v1/kb/{kbid}/slug/{rslug}",
        method="PATCH",
        path_params=("kbid", "rslug"),
```